packages/widget/src/components/AppContainer.tsx

```diff
@@ -64,9 +109,12 @@ export const AppContainer: React.FC<PropsWithChildren<{}>> = ({ children }) => {
         id={createElementId(ElementId.ScrollableContainer, elementId)}
         variant={variant}
         enableColorScheme
+        paddingTopAdjustment={positionFixedAdjustment}
         // ref={ref}
       >
-        <FlexContainer disableGutters>{children}</FlexContainer>
+        <FlexContainer disableGutters id="app-level-flex-container">
```

Comment:
Let's add this id attribute here please to be consistent https://github.com/lifinance/widget/blob/main/packages/widget/src/utils/elements.ts#L1

Reply:
I think can remove these - I've accidently left them in from debug the routes positioning so they aren't really needed